test/Riok.Mapperly.IntegrationTests/_snapshots/MapperTest.SnapshotGeneratedSource_NET8_0.verified.cs

```diff
@@ -588,9 +588,9 @@ private string MapToString(global::Riok.Mapperly.IntegrationTests.Models.TestEnu
         {
             return source switch
             {
-                global::Riok.Mapperly.IntegrationTests.Models.TestEnum.Value10 => nameof(global::Riok.Mapperly.IntegrationTests.Models.TestEnum.Value10),
-                global::Riok.Mapperly.IntegrationTests.Models.TestEnum.Value20 => nameof(global::Riok.Mapperly.IntegrationTests.Models.TestEnum.Value20),
-                global::Riok.Mapperly.IntegrationTests.Models.TestEnum.Value30 => nameof(global::Riok.Mapperly.IntegrationTests.Models.TestEnum.Value30),
+                global::Riok.Mapperly.IntegrationTests.Models.TestEnum.Value10 => "Value10",
```

Comment:
This changes removes the usage-references in dev tools. If the default strategy is used, I think it would be preferable to keep the `nameof`.
This maybe relates to my other comment proposing building the enum member mappings in the mapping builder instead of inside the mapping. This would allow to keep using `nameof` for the default naming strategy.

Reply:
Yes, similar to the other comment, I suggest waiting for the other PR to be merged before making this change.